src/components/admin/input-label/InputLabel.tsx

```diff
@@ -7,7 +7,7 @@ export interface InputLabelProps {
 }
 
 export const InputLabel = ({ htmlFor, text, isRequired }: InputLabelProps) => (
-    <label htmlFor={htmlFor} className="input-label">
+    <label htmlFor={htmlFor}>
         {isRequired && <span className="required-field">*</span>}
         {text}
     </label>
```

Comment:
Removing the "input-label" class breaks styles in the program modal (see screenshot). Please revert this change.

```suggestion
    <label htmlFor={htmlFor} className="input-label">
        {isRequired && <span className="required-field">*</span>}
        {text}
    </label>
```

<img width="1216" height="1034" alt="Image" src="https://github.com/user-attachments/assets/a895e030-a537-4e91-ac7d-3abbd318816d" />

Reply:
This issue was probably caused by automatic merge conflict resolve. Fixed!